smartsim/_core/_cli/cli.py

```diff
@@ -73,14 +74,34 @@ def execute(self, cli_args: t.List[str]) -> int:
             self.parser.print_help()
             return 0
 
-        app_args = cli_args[1:]
-        self.args = self.parser.parse_args(app_args)
+        app_args = cli_args[1:]  # exclude the path to executable
+        subcommand = cli_args[1]  # first positional arg is the subcommand
```

Comment:
Thanks for these comments! Made it much easier to figure out what the slice operation was meant for!!